mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -1307,3 +1307,90 @@ Feature: Loans tests
     When method POST
     Then status 422
     And match $.errors[0].message == blockMessage
+
+  Scenario: When patron has exceeded their Patron Group Limit for 'Recall overdue by maximum number of days', patron is not allowed to borrow items per Conditions settings
+
+    * def extUserBarcode1 = 'FAT-1024UBC-1'
+    * def extUserBarcode2 = 'FAT-1024UBC-2'
+    * def extItemBarcode1 = 'FAT-1024IBC-1'
+
+    # location and service point setup
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint')
+
+    # post an item
+    * def extItemId1 = call uuid1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(extItemId1), extItemBarcode: #(extItemBarcode1) }
+
+    # post a group and users
+    * def extUserId1 = call uuid1
+    * def extUserId2 = call uuid1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup') { extUserGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId1), extUserBarcode: #(extUserBarcode1), extGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId2), extUserBarcode: #(extUserBarcode2), extGroupId: #(groupId) }
+
+    # get the interested condition id, Recall overdue by maximum number of days
+    Given path 'patron-block-conditions'
+    When method GET
+    Then status 200
+    * def conditions = karate.sort(response.patronBlockConditions, (condition) => condition.name)
+    * def conditionId = conditions[5].id
```

Comment:
What if someday in the future we are going to have a new condition type? Maybe we should fine the needed condition in some other way?

Reply:
fixed